src/style.css

```diff
@@ -1,3 +1,72 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Avoid using the universal selector '*' for setting styles, as it can have performance implications. If you need to reset margins and paddings, consider specifying the elements that require this reset or using a more efficient CSS reset technique.